qr_generator/lib/utils/api_client.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:dio/dio.dart';
+
+class ApiClient {
+  late Dio dio;
+  ApiClient() {
```

Comment:
**fix**: prefer using `factory` constructors since they're meant to build new instances that process data before actually returning the instance.

```
final Dio dio;

const ApiClient._({
  required this.dio,
});

factory ApiClient() {
      final options = BaseOptions(
        connectTimeout: 5000,
        receiveTimeout: 5000,
        baseUrl: 'http://192.168.86.45:3030/', //192.168.86.30
      );

    return ApiClient._(
      dio: Dio(options),
    );
}

Reply:
Done!